bentoml/_internal/bento/build_config.py

```diff
@@ -606,6 +886,14 @@ def with_defaults(self) -> "FilledBentoBuildConfig":
 
     @classmethod
     def from_yaml(cls, stream: t.TextIO) -> "BentoBuildConfig":
+        """
+        Args:
+            stream (:code:`TextIO`):
+                A YAML file stream.
```

Comment:
If there's already a type annotation for the function, most of time we don't need document the "Args" here. unless it's user facing and we want to show it in the API reference

```suggestion
```